_layouts/academia/leic/pdm.md

```diff
@@ -1,15 +1,23 @@
 ---
 layout: course
 title:
-  en: 
-  pt: 
-repository: 
-isel: 
-year: 
-semester: 
-ects: 
+  en: Mobile Device Programming
+  pt: Programação em Dispositivos Móveis
+repository: https://github.com/isel-leic-pdm
+isel: https://isel.pt/disciplinas/programacao-em-dispositivos-moveis-leic-pn
+year: 3
+semester: 1
+ects: 6
 professor:
-  display_name: 
+  display_name: Paulo Pereira
 bibliography: |
-  Add your content here in Markdown
+  Selected articles and sewlected parts of the reference documentation of the targeted platform (currently Android)
```

Comment:
This part is common two both languages. This means that it appears in EN on the PT page.